src/components/Loaders/cardShimmer/index.js

```diff
@@ -1,15 +1,21 @@
-import React from 'react';
-import classNames from './cardShimmer.module.scss';
+import React from "react";
```

Comment:
single-quote and double-quote confusions, it is better to check and stick with the projects style guide. And not use your personal preference.
Check the eslint config for the project or you can refer other files

Reply:
Okay, I'll fix it.

Reply:
I've made the changes please check.